---

layout: default
title: Evgeniia Golovina - CV

---


## Evgeniia Golovina

<img align="right" src="images/Evgeniia.jpg">

A PhD candidate at the Liggins Institute, University of Auckland, New Zealand. 

Currently, I’m doing research in Justin O’Sullivan “Genomics and System Biology” group, where we’re trying to understand the role of genetics and epigenetics in the development of complex phenotypes. Particularly, I’m trying to understand how genetic variants associated with psychiatric and substance use disorders can act to regulate genes and how the knowledge on the 3D genome organization might inform our understanding of tissue- and cell type-specific gene regulation in the development of these disorders.


&nbsp;

**Summary:**

- NGS and multi-omics data analysis, 7 years
- Broad knowledge in biochemistry, genetics and molecular biology
- Programming experience in Python, R, Matlab, C++, Bash
- Excellent organisational, writing and communication skills
- High motivation to get things done
- Ability to work in team and independently


**Contact information:**

- Tel: +64 21 087 45 886
- Email: Jn.Golovina@gmail.com, e.golovina@auckland.ac.nz
- Linkedin: [https://www.linkedin.com/in/evgeniiagolovina/](https://www.linkedin.com/in/evgeniiagolovina/)
- Github: [https://github.com/Eugeniia](https://github.com/Eugeniia)
- Twitter: [https://twitter.com/FoffaJn](https://twitter.com/FoffaJn)
- ResearchGate: [https://www.researchgate.net/profile/Evgeniia_Golovina2](https://www.researchgate.net/profile/Evgeniia_Golovina2)
- Google Scholar: [https://scholar.google.ru/citations?user=cR7t9_8AAAAJ&hl=en&authuser=1](https://scholar.google.ru/citations?user=cR7t9_8AAAAJ&hl=en&authuser=1)


&nbsp;

### EDUCATION

**2017 &ndash; 2021**<br/>
**PhD**. *University of Auckland, Liggins Institute, Health Sciences*<br/>
PhD thesis “Decoding the regulatory landscape of psychiatric and alcohol use
disorders”<br/>

**2012 &ndash; 2014**<br/>
**MSc**. *Saint Petersburg State University, Biology*<br/>
Specialization: Biochemistry and Molecular Biology<br/>
MSc thesis “Improvement of information value of spectroscopic data in the analysis of biodeteriorations”<br/>

**2012 &ndash; 2014**<br/>
**Professional Communication Translator**. *Saint Petersburg State University, English*<br/>

**2008 &ndash; 2012**<br/>
**BSc**. *Saint Petersburg State University, Biology*<br/>
Specialization: Cell and Molecular Biology, Biotechnology<br/>
BSc thesis “Diagnostics and genotyping of viral hepatitis C in Saint Petersburg and Leningrad region in 2010-2011”<br/>


&nbsp;

### PROFESSIONAL & RESEARCH EXPERIENCE

<strong>10.12.2013 &mdash; 25.08.2017</strong><br/>
<strong>[Genestack](https://genestack.com/)</strong>, Saint Petersburg, Russia<br/>
Position: Biologist<br/>

Genestack is a platform to manage, process and visualize biological data, with a goal to help users leverage high-throughput multi-omics data and fast-track drug discovery, precision medicine and bioinformatics research in the post-genomic era. At Genestack I was involved in:

- NGS data analysis
- development of test system for biological applications
- curation of experiments pre-uploaded from public databases
- development of user friendly interfaces via coordination with software developers
- promotion of company products through writing tutorials and blog posts
- user support

&nbsp;

<strong>01.02.2013 &mdash; 26.09.2013</strong><br/>
<strong>Nevinpat</strong>, Saint Petersburg, Russia<br/>
Position: Patent agent<br/>

Nevinpat is a Saint Petersburg division of [Papula-Nevinpat](https://www.papula-nevinpat.com/papula-nevinpat/), one of the largest trademark, design and patent agencies in Finland. At Nevinpat I was involved in: 

- protection of foreign applications (Russian and Eurasian patent offices)
- editing Russian translations of patent applications
- correspondence with patent experts and foreign applicants
- comparative analysis of patentability
- defining patent infringement

&nbsp;

<strong>01.12.2010 &mdash; 29.06.2012</strong><br/>
<strong>[Smorodintsev Research Institute of Influenza](https://influenza.spb.ru/en/about_institute/institute_today/)</strong>, Saint Petersburg, Russia<br/>
Position: Research Assistant<br/>

Smorodintsev Research Institute of Influenza is one of the leading scientific centers tasked with handling urgent issues in virology, epidemiology, viral pathogenesis and clinical manifestations. At the institute, I was involved in laboratory diagnosis and investigation of hepatitis C (HCV), torque teno (TTV) and influenza viruses.

&nbsp;

### SKILLS & INTERESTS

**Personal qualities:**
Self-starter on exploratory projects, goal-driven, scientific curiosity, highly interested
in getting things done, excellent organisational and communication skills, eager to
learn and develop new skills, willing to travel up to 80%

**Data analysis and computer skills:**
Expertise with commonly used software applications and bioinformatics tools for analysis of NGS and multi-omics data (Hi-C, RNA-seq, WGS, WES, WGBS, RRBS, single-cell RNA-seq, microarray, ChIP-seq, DNase-seq, microbiome) and spectroscopic data (Raman and Infrared spectroscopy); experience with NeSI, Nectar Research Cloud, eResearch VMs and Google Cloud computing platforms; familiarity with Docker/Singularity, database administration tools (e.g. PostgreSQL), GitHub, Jupyter Notebook, Spyder, PyCharm and TensorFlow

**Programming skills:**
Python, R, MatLab, C++, Bash

**Languages:**
Russian (native proficiency), English (professional working proficiency), Spanish (limited working proficiency), German (elementary proficiency)

**Professional interests:**
Bioinformatics, data science and visualization, data storytelling, public speaking, machine learning, 4D genome architecture and gene regulation, psychiatric genetics, precision medicine, innovation and commercialization in science


&nbsp;

### ACTIVITIES

- **Hi-C workshop** by Genomics Aotearoa, 25-26th March 2021, Auckland, NZ<br/>
- Organization of the **Student Research Day at Liggins Institute**, 1st December 2020, Auckland, NZ<br/>
- **Virtual EMBL Conference: From Functional Genomics to Systems Biology** (digital poster), 16-19
November 2020<br/>
- **16th Asian Conference on Transcription 2019** (poster presentation), 1-4 December 2019, Dunedin, NZ<br/>
- **Liggins Institute public lecture** “Precision medicine: your genes and mental health”, 26th November 2019, Auckland, NZ. Available [here](https://www.youtube.com/watch?v=ZdHQQR4Y_vU&feature=emb_logo%2F)<br/>
- **Queenstown Research Week 2019** (oral presentation), 31 August - 6 September 2019, Queenstown, NZ<br/>
- **HealtheX 2019 conference** (oral presentation), 13th September 2019, Auckland, NZ<br/>
- **The Research Bazaar (ResBaz) 2019**, helper for “Introductory R” session, 10-12 July 2019, Auckland, NZ<br/>
- **HealtheX 2018 conference** (elevator pitch presentation), 7th September 2018, Auckland, NZ<br/>
- **Winter School in Mathematical & Computational Biology** by the University of Queensland, 2-6 July 2018, Brisbane, Australia<br/>
- **Enterprise & Innovation Workshop** by Centre for Innovation and Entrepreneurship at the University of Auckland, 7 & 13 March 2018, Auckland, NZ<br/>
- **PhD Research Innovation & Commercialisation Course** by Centre for Innovation and Entrepreneurship at the University of Auckland, 9-10 November 2017, Auckland, NZ<br/>

&nbsp;

### PUBLICATIONS

- **Golovina E**, Fadason T, Lints T, Walker C, Vickers MH, O'Sullivan JM. Autism spectrum disorder: understanding the impact of SNPs on biological pathways in the fetal and adult cortex. bioRxiv. 2021:2021.03.03.433667. [DOI](https://doi.org/10.1101/2021.03.03.433667)
- **Golovina E**, Vickers MH, Erb CD, O’Sullivan JM. GWAS SNPs impact shared regulatory pathways amongst multimorbid psychiatric disorders and cognitive functioning. Front Psychiatry. 2020;11:1079. [DOI](https://www.frontiersin.org/articles/10.3389/fpsyt.2020.560751/full)
- Hibberd R, **Golovina E**, Farrow S, O’Sullivan JM. Genetic variants associated with alcohol dependence coordinate regulation of ADH genes in gastrointestinal and adipose tissues. Sci Rep. 2020;10:1-10. [DOI](https://www.nature.com/articles/s41598-020-66048-z)
- Farrow S, Schierding W, Gokuladhas S, **Golovina E**, Fadason T, Cooper AA, O’Sullivan JM. Establishing gene regulatory networks from Parkinson's disease risk loci. bioRxiv. 2021.04.08.439080. [DOI](https://doi.org/10.1101/2021.04.08.439080)
- Gokuladhas S, Schierding W, **Golovina E**, Fadason T, O’Sullivan JM. Unravelling the shared genetic mechanisms underlying 18 autoimmune diseases using a systems approach. bioRxiv. 2021.03.28.437437. [DOI](https://doi.org/10.1101/2021.03.28.437437)
- Fadason T, Gokuladhas S, **Golovina E**, Ho D, Farrow S, Nyaga D, et al. A transcription regulatory network within the ACE2 locus may promote a pro-viral environment for SARS-CoV-2 by modulating expression of host factors. bioRxiv. 2020:2020.04.14.042002. [DOI](https://www.biorxiv.org/content/10.1101/2020.04.14.042002v2)

&nbsp;

### ADDITIONAL EDUCATION

- [Docker Masterclass for Machine Learning and Data Science](https://www.udemy.com/certificate/UC-c2ac9cbc-46c0-4312-8271-6fb7727c991c/) (Udemy)
- [Psychological First Aid](http://coursera.org/verify/PPJQDEAVY5AK) by Johns Hopkins University (Coursera)
- Data Science Specialization by Johns Hopkins University (Coursera)
- Machine Learning A-Z™: Hands-On Python & R In Data Science (Udemy)
- Communication Strategies for a Virtual Age by University of Toronto (Coursera)
- Strategies for Speaking, Teaching and Presenting Online by Public Speaking Lab
- Mathematics for Data Science Specialization by Higher School of Economics (Coursera)
- [Machine Learning](https://www.coursera.org/api/legacyCertificates.v1/spark/statementOfAccomplishment/971489~2042534/pdf) by Stanford University (Coursera)

&nbsp;

### REFEREES

- **Prof. Justin O’Sullivan**, *Liggins Institute, University of Auckland*, email: justin.osullivan@auckland.ac.nz
- **Prof Emer. John Werry**, *University of Auckland*, email: j.werry@auckland.ac.nz
- **MBA, PhD. Alessandro Riccombeni**, *Amazon Web Services*, email: alessandro.riccombeni@gmail.com
- **Prof. Lyudmila Panina**, *Saint Petersburg State University*, email: lkpanina@yandex.ru


<div align="right"><i>Best printed with Google Chrome</i></div>
